editor/plugins/sprite_frames_editor_plugin.cpp

```diff
@@ -458,6 +471,9 @@ void SpriteFramesEditor::_sheet_spin_changed(double p_value, int p_dominant_para
 			const Size2i count = (size + separation) / (frame_size + separation);
 			split_sheet_h->set_value(count.x);
 			split_sheet_v->set_value(count.y);
+
+			split_data["dom_x"] = int(split_sheet_h->get_value());
+			split_data["dom_y"] = int(split_sheet_v->get_value());
```

Comment:
This is now reversed, when `PARAM_SIZE` you store the size, and calculate the other value, and vice versa